test/e2e/SKIPPED_TESTS.yaml

```diff
@@ -1,8 +1,12 @@
 skipped_tests:
 # CloudStack
 - TestCloudStackKubernetes123RedhatProxyConfig
+- TestCloudStackKubernetes124RedhatProxyConfig
```

Comment:
why skip those tests?

Reply:
Proxy config isn't set up for Cloudstack CI env, which is why the 123 tests were skipped currently ?